spec/controllers/partners_controller_spec.rb

```diff
@@ -2,14 +2,7 @@
 
 describe PartnersController, type: :controller, include_shared: true do
   context "when authenticated" do
-    login_partner
-
-    describe "GET #new" do
```

Comment:
This test was removed as this flow does not exist in our application. (Partners are currently created via devise_invitable. This flow was kept the same but might be worth changing in the future. For now the test was deleted as it was testing something we didnt want to test)